app/test_main.py

```diff
@@ -1,3 +1,20 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_function_should_return_list() -> None:
+    assert isinstance(get_human_age(0, 0), list)
+
+
+def test_should_return_list_size_2() -> None:
+    assert len(get_human_age(0, 0)) == 2
+
+
+def test_should_return_correct_parametrize() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+    assert get_human_age(14, 14) == [0, 0]
+    assert get_human_age(15, 15) == [1, 1]
+    assert get_human_age(23, 23) == [1, 1]
+    assert get_human_age(24, 24) == [2, 2]
+    assert get_human_age(27, 27) == [2, 2]
+    assert get_human_age(28, 28) == [3, 2]
+    assert get_human_age(100, 100) == [21, 17]
```

Comment:
Consider using `@pytest.mark.parametrize` to avoid repeating similar test cases. This will make your tests cleaner and more maintainable, as suggested in the checklist: 'Use @pytest.mark.parametrize'.